samples/apps/agent-id-sample/README.md

```
@@ -106,28 +156,30 @@ In four terminals:
+cd ai-agent && npm install && npm start                   # http://localhost:8790/chat
 cd frontend && npm install && npm run dev                 # http://localhost:5173
 ```
 
 `npm run seed` initializes the local SQLite database with sample flights, hotels, and trips. Run it once on first setup.
 
 ## Try It
 
-Open `http://localhost:5173`, open the chat widget, and try:
+Open `http://localhost:5173`, sign in as `john.doe` / `john.doe`, open the chat widget, and try:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)